tests/TestCase/I18n/MessagesFileLoaderTest.php

```diff
@@ -88,12 +88,12 @@ public function testTranslationFoldersSequence(): void
 
         $expected = [
             ROOT . DS . 'tests' . DS . 'test_app' . DS . 'resources' . DS . 'locales' . DS . 'en_' . DS,
```

Comment:
Looking under `en_` doesn't make sense. We should skip using the value generated by `implode('_', [$locale['language'], $locale['region']]);` if the `region` is empty.